can_tools/scrapers/official/federal/HHS/facility.py

```diff
@@ -105,6 +105,21 @@ def normalize(self, data: str) -> pd.DataFrame:
         temp = temp + df.eval(f"{pps} / {ppc}").fillna(0.0)
         df.loc[:, "inpatient_beds_used_covid_7_day_canavg"] = temp.values
 
+        # Calculate adult and pediatric hospital admissions (same process as above).
+        adult_admissions = "previous_day_admission_adult_covid_confirmed_7_day_sum"
```

Comment:
I've copied the averaging done above here, but is it possible that we might want to instead collect the raw `sum` of admissions, to try to match the CDC's indicator of `New COVID-19 admissions per 100,000 population (7-day total)`?

Reply:
Hrm, interesting...  I *think* this code makes sense as-is, but we'll definitely want to at minimum sanity-check it against the CDC data.

My understanding is that hospitals have a "survey" to fill out, ideally daily, and one of the questions is basically "How many confirmed covid admissions did you have in the previous day?" and the _sum column adds those answers up over 7 days. BUT they might not fill it out every day, so the _coverage number could be 0-7 indicating how many times they actually filled it out.  To get a valid estimate for the week, I think we *have* to take sum/coverage, else we would end up with lower sums for hospitals that report less frequently.  

I don't 100% trust myself though (notably the metrics we did this for before were "current" metrics, not "new" metrics so having <7 samples is less of a problem).  So please do try to sanity-check this in any way that makes sense to you.

Reply:
I see, thanks for this context 🙏 🙏 

It seems that the CDC must be using the `_sum` data, or some variation of it based on their values. I'm not sure if they're concerned about hospitals not reporting every day (and if they aren't, why not) or if they are accounting for this somehow.

Our values do not quite line up 1:1 even with this disaggregation (though they are usually close), so I'm in the process of hashing out why that might be.